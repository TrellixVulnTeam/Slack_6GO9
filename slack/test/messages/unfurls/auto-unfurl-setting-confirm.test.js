const AutoUnfurlSettingConfirm = require('../../../lib/messages/unfurls/auto-unfurl-setting-confirm');

describe('AutoUnfurlSettingConfirm message rendering', () => {
  test('works for any channel', async () => {
    const message = new AutoUnfurlSettingConfirm('integrations/snappydoo', true);
    expect(message.getAttachment()).toMatchSnapshot();
  });
  test('works for one channel', async () => {
    const message = new AutoUnfurlSettingConfirm('integrations/snappydoo', false);
    expect(message.getAttachment()).toMatchSnapshot();
  });
});
