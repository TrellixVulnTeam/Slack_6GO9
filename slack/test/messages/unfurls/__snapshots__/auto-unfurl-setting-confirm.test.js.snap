// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutoUnfurlSettingConfirm message rendering works for any channel 1`] = `
Object {
  "color": "#24292f",
  "mrkdwn_in": Array [
    "text",
  ],
  "text": "Links to \`integrations/snappydoo\` that you post in any channel of this workspace will automatically have a rich preview :white_check_mark:",
}
`;

exports[`AutoUnfurlSettingConfirm message rendering works for one channel 1`] = `
Object {
  "color": "#24292f",
  "mrkdwn_in": Array [
    "text",
  ],
  "text": "Links to \`integrations/snappydoo\` that you post in this channel will automatically have a rich preview :white_check_mark:",
}
`;
